zerver/lib/user_status.py

```diff
@@ -22,16 +26,18 @@ class RawUserInfoDict(TypedDict):
     emoji_name: str
     emoji_code: str
     reaction_type: str
+    scheduled_end_time: datetime | None
 
 
-def format_user_status(row: RawUserInfoDict) -> UserInfoDict:
+def format_user_status(user_id: int, row: RawUserInfoDict) -> UserInfoDict:
```

Comment:
Maybe call this `acting_user_id`, since that's terminology we use to indicate that the user ID specifically refers to the current acting user.